aten/src/ATen/native/native_functions.yaml

```diff
@@ -3246,14 +3246,14 @@
   dispatch:
     CompositeExplicitAutograd: mvlgamma_
 
-- func: narrow_copy(Tensor self, int dim, int start, int length) -> Tensor
+- func: narrow_copy(Tensor self, int dim, int start, SymbolicOrConcreteInt length) -> Tensor
```

Comment:
Can we call this something shorter haha. Maybe SizeInt? (Size might be bad because torch.Size is a thing and represents multiple sizes together)